Codeinwp/otter-blocks
# Update dependancies

Closes #1667 


### Summary


💯  Update NPM dependancies. We now use the latest dev packages of WordPress

⚠️ Node 18 has been set as default version. `package-lock.json` is now at version `3`.

🧪  E2E via Playwright are now available.

Other:
- Performance test has been update.
- Block ID has been refactored. Now it checks the id after duplication and inserting.
- New tests made in Playwright
  - Progress Bar: just checking if we can insert it.
  - Form: checking for inserting and option saving.
 

### Screenshots 

Playwright development UI where you can interact with the tests. `npm run test:e2e:playwright-ui` 

<img width="1498" alt="Screenshot 2023-05-23 at 13 50 18" src="https://github.com/Codeinwp/otter-blocks/assets/17597852/4f2e0364-c145-4d4e-92d3-3c70038e5bcb">



----

### Test instructions


This is an update on development tools. No feature added or bug fixed, just general test to make sure plugin is working fine and nothing breaks.



---- 

### Checklist before the final review
